include/dts-functions.sh

```diff
@@ -1387,3 +1387,92 @@ show_menu() {
     echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to launch SSH server  ${NORMAL}"
   fi
 }
+
+check_if_fused() {
+  file_path="/sys/class/mei/mei0/fw_status"
+
+  if [[ ! -f $file_path ]]; then
+      echo "File not found: $file_path"
+      return 2
```

Comment:
This indent has 6 spaces instead of 4 as in other places in this script.

Reply:
0548c9b9045e4fb30addeb8645a0c353d11c8082